app/locales/en/translations.js

```diff
@@ -305,9 +305,9 @@ export default {
         heading: 'Withdraw {{documentType.singularCapitalized}}',
         withdrawal_form_heading: 'Reason for withdrawal (required):',
         pre_moderation_notice_pending: 'Your {{documentType.singular}} is still pending approval and thus private, but can be withdrawn immediately. If you wish to provide a reason for withdrawal, it will be displayed only to service moderators. Once withdrawn, your preprint will never be made public.',
-        pre_moderation_notice_accepted: 'This service uses pre-moderation. Your {{documentType.singular}} withdrawal must be approved by a moderator. Once approved, your {{documentType.singular}} will be removed, but basic metadata (like title, authors, and reason for withdrawal, if provided) will remain.',
-        post_moderation_notice: 'This service uses post-moderation. Because of that, your request will need to be approved by a moderation administrator before your {{documentType.singular}} can be withdrawn. If you request is approved, your {{documentType.singular}} will be replaced with a tombstone page with metadata and the reason for withdrawal (if included). Your {{documentType.singular}} will still be searchable by other users.',
-        no_moderation_notice: 'This request will be submitted to support@cos.io for review and removal. Upon removal, this {{documentType.singular}} will be replaced with a tombstone page with metadata and the reason for withdrawal (if included). This {{documentType.singular}} will still be searchable by other users after removal.',
+        pre_moderation_notice_accepted: '<strong>{{documentType.plural}} are a permanent part of the scholarly record. Withdrawal requests are subject to this service’s policy on {{documentType.singular}} removal and at the discretion of the moderators.</strong><br>This service uses post-moderation. This request will be submitted to service moderators for review. If the request is approved, this {{documentType.singular}} will be replaced by a tombstone page with metadata and the reason for withdrawal. This {{documentType.singular}} will still be searchable by other users after removal.',
```

Comment:
Is the first letter automatically capitalized by css? If not, do they need to be capitalized?

Reply:
can do `documentType.pluralCapitalized`, i think